components/popconfirm/popconfirm.ts

```diff
@@ -203,8 +207,11 @@ export class NzPopconfirmDirective extends NzTooltipBaseDirective {
                     (click)="onConfirm()"
                     [attr.cdkFocusInitial]="nzAutoFocus === 'ok' || null"
                   >
-                    <ng-container *ngIf="nzOkText">{{ nzOkText }}</ng-container>
-                    <ng-container *ngIf="!nzOkText">{{ 'Modal.okText' | nzI18n }}</ng-container>
+                    @if (nzOkText) {
+                      <ng-container>{{ nzOkText }}</ng-container>
```

Comment:
It looks like the <ng-container>s are no longer needed here, so let's remove them :)